views/js/runner/plugins/tools/lineReader/compoundMask.js

```diff
@@ -435,13 +449,24 @@ define([
             });
         }
 
+
         /**
          * =================================
          * Transform model related functions
          * =================================
          */
 
         function applyTransforms() {
+
+            // round dimensions and positions up to the next even integer
+            _.forOwn(dimensions, function(value, key) {
+                dimensions[key] = ceilToEvenInt(value);
+            });
+
+            _.forOwn(position, function(value, key) {
+                position[key] = ceilToEvenInt(value);
+            });
+
```

Comment:
I'm not sure about this. While the fix does seems to improve the zoom, I have the feeling it has some side effects like this one, that I can reproduce fairly easily now, while I cannot on develop (see the inner top right corner)

![screen shot 2017-05-12 at 10 31 02](https://cloud.githubusercontent.com/assets/11467480/25990026/91a0d3a4-36fe-11e7-9603-16feb1a679d6.png)